Yoast/javascript
# Remove KeywordInput margin

## Summary


  * Package(s) involved: yoast-components

  * Should the change be included in the package changelog?
  * Should the change be included in one or more plugin changelogs?
  * Package changelog item (if applicable): Removes the margin from the `KeywordInput` component.
  * Plugin changelog item (if applicable): 

  * Package(s) involved: __apps__/components

  * Should the change be included in the package changelog?
  * Should the change be included in one or more plugin changelogs?
  * Package changelog item (if applicable): 
  * Plugin changelog item (if applicable): 

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Plugin test instructions are here https://github.com/Yoast/wordpress-seo/pull/12932 and https://github.com/Yoast/wordpress-seo-premium/pull/2390
* Standalone: please check if the margin compensated in the example: see the keyword tab.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://github.com/Yoast/wordpress-seo/issues/12917
